usaspending_api/etl/management/commands/combine_universal_transaction_matview_chunks.py

```diff
@@ -0,0 +1,200 @@
+import logging
+import asyncio
+import sqlparse
+
+from django.db import connection
+from django.core.management.base import BaseCommand
+from usaspending_api.common.data_connectors.async_sql_query import async_run_creates
+from usaspending_api.common.helpers.timing_helpers import ConsoleTimer as Timer
+
+logger = logging.getLogger("script")
+
+RECREATE_TABLE_SQL = """
+DROP {old_object_type} IF EXISTS universal_transaction_matview_temp;
+CREATE TABLE universal_transaction_matview_temp AS SELECT * from universal_transaction_matview_0 WITH NO DATA;
+"""
+
+INSERT_INTO_TABLE_SQL = """
+INSERT INTO universal_transaction_matview_temp SELECT * FROM universal_transaction_matview_{current_chunk}
+"""
+
+TABLE_INDEX_SQL = """
+CREATE UNIQUE INDEX idx_ut_transaction_id_temp ON universal_transaction_matview_temp USING BTREE(transaction_id ASC NULLS LAST) WITH (fillfactor = 97);
```

Comment:
Not a fan of specifying the DDL of the table in separate files. A management command wouldn't be my first guess. I can see people forgetting where the indexes are defined.

Reply:
What's the alternative @tony-sappe? 

Or maybe you can expand upon the comment? 
- Is it that the DDL went from being stored in a JSON file to a Python file? If so, I kindof like that and have always thought managing SQL strings in JSON was really tough.
- Or is it that this DDL in this python file is one part, and there's more DDL in another Python (or JSON) file? -- disclosure: haven't done a full review of the code yet

Reply:
My comment is about parts of the matview DDL being stored in separate files not located near each other in the codebase.